contents/tutorials/delayed-survey.md

```diff
@@ -0,0 +1,66 @@
+---
+title: How to show a survey after a delay
+date: 2023-11-01
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+tags: ['surveys']
+---
+
+Before asking users about your app, you want them to **actually use it**. Surveys that pop up right away hinder their initial experience and get less useful responses as a result.
+
+To help you do prevent this, you can set up a delay surveys and this tutorial shows you how. It goes over how to create a survey targeting a HTML class in PostHog and then implement a delay using that class and survey in your app.
+
+## Create your delayed survey
+
+First, we create our class-targeted survey in PostHog by going to the [Surveys tab](https://app.posthog.com/surveys) and clicking "New survey." Choose any template or create a blank survey, fill out the details, and open the "Targeting" section. 
```

Comment:
"class targeted survey" is a confusing phrase. I think if you add the opening paragraph as mentioned above (explaining the approach), it will be easier to follow

```suggestion
First, we create our survey in PostHog by going to the [Surveys tab](https://app.posthog.com/surveys) and clicking "New survey." Choose any template or create a blank survey, fill out the details, and open the "Targeting" section. 
```